### DISCUSSION TITLE
Routing

### DISCUSSION THREAD

**Discussion by DeathlyBower959** [facebook/docusaurus#6638] [Q&A] [answered]
I'm not sure if this is the right place to ask, but you guys have exactly what I want.

- I want to use `BrowserRouter` (Just so I don't have the "#" in the URL)
- Hosting on GitHub pages

I have a react application using `react-router-dom`
I don't want to use Hash Router because of ugly URLs, so how did you manage to have clean URLs, which don't crash on reload?

I understand if this is the incorrect place to ask, but I'm not sure where else to inquire about this, because all tutorials that I have seen are outdated, or don't work for my case.

**Comment by Josh-Cena** [comment, marked as answer]
Responded on Discord.
